packages/clerk-js/src/ui/common/withRedirectToHome.tsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { useCoreSession, useEnvironment } from 'ui/contexts';
 import { useNavigate } from 'ui/hooks';
 
-export function withRedirectToHome<P extends SignInProps | SignUpProps>(
+export function withRedirectToHome<P extends SignInProps | SignUpProps | { continueExisting?: boolean }>(
```

Comment:
@yourtallness please check my comment below about removing `{ continueExisting?: boolean }` prop.

Reply:
Which comment are you referring to?

Reply:
https://github.com/clerkinc/javascript/pull/186#discussion_r849576512